domain/include/cstone/sfc/box.hpp

```diff
@@ -113,11 +113,12 @@ class Box
 {
 
 public:
-    HOST_DEVICE_FUN constexpr Box(T xyzMin, T xyzMax, BoundaryType b = BoundaryType::open)
```

Comment:
The FBC thickness is a detail of your current implementation of fixed boundaries. It should not be stored in the box or be part of a public interface.